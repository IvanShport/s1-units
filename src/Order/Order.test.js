import React from 'react'
import Order from './Order';
import {fakeOrders} from '../mock/fakeOrders';
import Enzyme, {shallow, mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('Order.getItems function', () => {

    const fakeOrder = {
        id: 123,
        date: 1544356800000,
        shop: 'Ali Express',
        items: [
            'Утиный пластмасса для показ новый год',
            'Курица из нержавеющей стали, утка, гусь, голубь, питьевой фонтан',
            'Новый стиль один розница яйцо для упаковки форма латекс',
        ]
    };

    const testingComponent = shallow(
        <Order order={fakeOrder}></Order>,
    );

	it('some valid order', () => {
    
        expect(testingComponent).toMatchSnapshot();
    });

    it('no items', () => {
        const props = {
            order: {
                id: 123,
                date: 1544356800000,
                shop: 'Ali Express',
            },
        }

        testingComponent.setProps(props);
    
        expect(testingComponent).toMatchSnapshot();
    });
});

