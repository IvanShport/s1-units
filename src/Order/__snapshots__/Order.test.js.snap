// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order.getItems function no items 1`] = `ShallowWrapper {}`;

exports[`Order.getItems function some valid order 1`] = `ShallowWrapper {}`;
